### DISCUSSION TITLE
Problems with traffmonetizer and packetstream

### DISCUSSION THREAD

**Discussion by dorel14** [MRColorR/money4band#20] [Q&A] [answered]
Hello
These 2 app don't share anything , i have set everything in .env file , it had work for a week and with latest version it doesn't work anymore

**Comment by MRColorR** [comment]
Hello,
Thank you for reaching out

-  Did you upgrade from a previous release? If yes, did the apps work
properly before?

- Can you please post the container logs of the affected apps? Also let me
know which OS are you using and wich docker version. Thanks

**Comment by MRColorR** [comment, marked as answer]
> Hello These 2 app don't share anything , i have set everything in .env file , it had work for a week and with latest version it doesn't work anymore

Today i've cheched my personal instance on docker for Windows and found out:

- Traffmonetizer endpoint went down on 11-dec-2022 with the following error _"Resource temporarily unavailable"_ , today it seems to be up and running again as per log message _"Connected: True; Accepting: True"_. So i think it's been a traffmonetizer outage
- Packetstream also had an outage as stated in logs from 11-dec-200 to 12-dec-2022 with _Unable to connect to validation server." error_ and then from 12-dec-2022 to 13-dec-2022 with the following error _"Error communicating with PacketStream server."_ now it seems to be working again (as per log message _""ok":true,"message":"PacketStream background process is running.""_. 

If they're back online for you too good, if not restart your machine and also check your DNS settings. if errors persist let me know, I'm here to help you.

- [x] Money4band is capable of auto-restart after errors, self updating and self healing so for non code related issues the best option is just check logs, see here on github discussions/issues if someone had same issue and check on app's socials pages if there are outages on their side.
